Zend/tests/partial_application/variation_debug_001.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+Partial application variation debug user
+--FILE--
+<?php
+function bar($a = 1, $b = 2, ...$c) {
+
+}
+
+var_dump(bar(?, new stdClass, 20, new stdClass, four: 4));
```

Comment:
I'm not sure of the best way to represent this in debug info. Very open to suggestions, the output doesn't always make sense right now, I figured this would be a detail we could perfect ... when we know if anyone is going to be debugging it ;)